core/src/main/java/org/elasticsearch/plugins/PluginCli.java

```diff
@@ -40,7 +39,20 @@ public PluginCli(Environment env) {
     }
 
     public static void main(String[] args) throws Exception {
-        BasicConfigurator.configure(new NullAppender());
+        // initialize default for es.logger.level because we will not read the logging.yml
```

Comment:
> We could also fire the logs through a log4j appender that targets our Terminal object.

This is what I would expect, otherwise as it is right now we make an arbitrary decision on log level, while we should be setting it to DEBUG and letting the Terminal filter based on Verbosity.

However, what you've added here is exactly as it was before I removed this a while back, so this can be a spin off issue we can handle whenever someone can get to it.